http/src/main/resources/reference.conf

```diff
@@ -1228,7 +1228,7 @@ autoFreeze {
 }
 
 autodelete {
-  autodeleteCheckInterval = 2 hours
+  autodeleteCheckInterval = 5 minutes
```

Comment:
let me know if you think this would be too resource-intensive.  "significantly less than 1 hour" is my goal, so we can be flexible here